cooked-validators/src/Cooked/Tx/Constraints/Type.hs

```diff
@@ -124,34 +126,49 @@ class ToOutputDatum a where
 instance ToOutputDatum Pl.OutputDatum where
   toOutputDatum = id
 
+instance ToOutputDatum Pl.Datum where
+  toOutputDatum = Pl.OutputDatum
+
 instance ToOutputDatum () where
   toOutputDatum = const Pl.NoOutputDatum
 
 instance ToOutputDatum Pl.DatumHash where
   toOutputDatum = Pl.OutputDatumHash
 
-data ConcreteOutput ownerType datumType where
+class ToValue a where
+  toValue :: a -> Pl.Value
+
+instance ToValue Pl.Value where
+  toValue = id
+
+instance ToValue Pl.Ada where
+  toValue = Pl.toValue
```

Comment:
Now I'm not sure — if these two instances are all that's ever gonna be, why do we want to parametrize `IsOutput` over values?